module/platform/darwin/HardwareIO/data/config/HardwareIO.yaml

```diff
@@ -1,7 +1,7 @@
 battery:
-  charged_voltage: 12.6
```

Comment:
Why are these changes in this PR?

Reply:
The Darwin voltages were overwritten when copy & pasting, will revert